pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -144,7 +144,7 @@ func GetQueueJobKey(obj interface{}) (string, error) {
 // UpdateQueueJobStatus was part of pod informer, this is now a method of queuejob_controller file.
 // This change is done in an effort to simplify the controller and enable to move to controller runtime.
 func (qjm *XController) UpdateQueueJobStatus(queuejob *arbv1.AppWrapper) error {
-	labelSelector := fmt.Sprintf("%s=%s", "appwrapper.mcad.ibm.com", queuejob.Name)
+	labelSelector := fmt.Sprintf("%s=%s", "ray.io/cluster", queuejob.Name)
```

Comment:
This look suspicious.
Why would MCAD label pod using Ray label?

Reply:
Right. @ChristianZaccaria this probably works for your particular use case, but that mechanism must be generic, for whatever resources managed by the AppWrapper. So the root cause is likely a missing update of these managed resources, so they are labeled with `appwrapper.mcad.ibm.com`.

Reply:
I think that makes sense now. The managed resources are not updating their labels to use `appwrapper.mcad.ibm.com`. So, are we speaking about a bug issue, or is this something just happening on my end? (I suppose in any case, we could probably close this issue). Thanks!

Reply:
It's very possible it's a bug. One possibility is that controller owning the resource / generic item overwrites the label applied by MCAD.